tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrNotAllowedNode.hlsl

```diff
@@ -0,0 +1,12 @@
+// RUN: %dxc -T lib_6_7 %s | FileCheck %s
+// note, this test would pass if the node shader
+// launch type was NodeLaunchType::Thread
+
+
+// CHECK: error: Node shader 'N' with broadcasting launch type requires 'numthreads' attribute
```

Comment:
This doesn't look to be the error you wanted to test here. Additionally, I would expect lib_6_7 to fail eventually for lack of node shaders.